lib/routes/githubAuth.js

```diff
@@ -0,0 +1,12 @@
+Router.route('/github/authenticate', {
+  name: 'github.authenticate',
+
+  waitOn() {
+    Meteor.subscribe('gitAuth');
+  },
+
+  action() {
+    this.render('githubAuth');
+  },
+
+});
```

Comment:
This route should be for admin only, add to the `OnBeforeAction()` hook [here](https://github.com/desoukya/SEMS/blob/master/lib/routes/hooks.js)